app/services/commitments/calculate_amount_service.rb

```diff
@@ -23,27 +23,10 @@ def call
     def commitment_amount_cents
       return 0 if !commitment || !invoice_subscription || commitment.amount_cents.zero?
 
-      # No proration when subscription is anniversary
-      return commitment.amount_cents if subscription.anniversary?
+      service = Commitments::HelperService.new(commitment:, invoice_subscription:, current_usage: false)
+      result = service.proration_coefficient
```

Comment:
can you please use another variable and not result since we would mix it with the result instance variable from the BaseService (that is also parent class here)